src/ReactTelephoneInput.js

```diff
@@ -0,0 +1,509 @@
+'use strict';
```

Comment:
This file is moved from `index.js`. The only modification is the removal of the `require('../less/default.less');` — everything else is the same.